### PR TITLE
feat(api-keys): add CRUDL

### PR DESCRIPTION
**Pull request by kapetr**
https://github.ibm.com/Incubation/bee-ui/issues/616

Blocked by API: 
- show key owners
- disable regenerate on keys of other owners

### CODE REVIEW

File: src/modules/api-keys/manage/ApiKeyModal.tsx [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+          ) : (
+            'Generate key'
+          )}
+        </Button>
+      </ModalFooter>
+    </Modal>
+  );
+}
+
+ApiKeyModal.Regenerate = function ViewModal({
```
Comment (PetrBulanek) [inline src/modules/api-keys/manage/ApiKeyModal.tsx:189]: `function RegenerateModal`

File: src/modules/api-keys/manage/ApiKeyModal.tsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+        </p>
+      </ModalHeader>
+      <ModalBody>
+        <ApiKeyDetail apiKey={apiKey} isSecretVisible />
+      </ModalBody>
+    </Modal>
+  );
+};
+
+ApiKeyModal.Delete = function ViewModal({
```
Comment (PetrBulanek) [inline src/modules/api-keys/manage/ApiKeyModal.tsx:248]: `function DeleteModal`

File: src/styles/global.scss
Comment (PetrBulanek) [inline src/styles/global.scss:1]: Not in scope of this PR, but this starts to get bigger, so we could maybe split it into per component files.

**Review by PetrBulanek — APPROVED** [review]

File: src/layout/shell/UserProfile.tsx [hunk 3]
```diff
@@ -62,7 +64,7 @@ export function UserProfile() {
       [
         {
           label: 'Preferences',
-          onClick: () => openModal((props) => <UserSettingsModal {...props} />),
```
[the remark this answers is not in the document]
Comment (kapetr) [reply src/layout/shell/UserProfile.tsx:65]: Yes, let's delete it.

File: src/styles/global.scss
Comment (kapetr) [reply src/styles/global.scss:1]: Agree

**Pull request merged**